src/main/java/com/epam/reportportal/extension/azure/AzureExtension.java

```diff
@@ -119,6 +139,8 @@
 
 	private final Supplier<EntityService> entityServiceSupplier;
 
+	private final List<AttachmentInfo> attachmentsURL = new ArrayList<>();
```

Comment:
not sure it's a good idea to have it as a member. what if several requests comes to one instance? better to pass it in functions

Reply:
I agree.  Made it a local variable.